recipes/ncbi-cxx-toolkit-public/26.0/conanfile.py

```diff
@@ -0,0 +1,1532 @@
+from conans import ConanFile, CMake, tools
+from conans.errors import ConanException
+from conans.errors import ConanInvalidConfiguration
+import os
+import shutil
+import subprocess
+import tempfile
+import sys
+import stat
+
+def _remove_readonly(fn, path, excinfo):
+    try:
+        os.chmod(path, stat.S_IWRITE)
+        fn(path)
+    except Exception as exc:
+        print("Skipped: ", path, " because ", exc)
+
+class NcbiCxxToolkit(ConanFile):
+    name = "ncbi-cxx-toolkit-public"
+    license = "CC0-1.0"
```

Comment:
https://github.com/ncbi/cxx-toolkit/blob/1966425d783aa586caa424bab0d2ad411f8ed42b/LICENSE#L24
Maybe?

```suggestion
    license = "Unlicense"
```

Reply:
This is not up to you or me. I put there what my organization tell me to.